ReportingServicesTools/Functions/CatalogItems/Set-RsDataSource.ps1

```diff
@@ -53,7 +53,7 @@ function Set-RsDataSource
             This command will establish a connection to the Report Server located at $rsProxy using current user's credentials and update the details of data source found at '/path/to/my/datasource'.
     #>
     
-    [cmdletbinding()]
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
```

Comment:
This was missing from the cmdlet binding, please verify the `ConfirmImpact` value